crates/autopilot/src/infra/solvers/dto/settle.rs

```diff
@@ -11,6 +11,8 @@ pub struct Request {
     /// Unique ID of the solution (per driver competition), to settle.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     pub solution_id: u64,
+    /// The last block number in which the solution TX can be included
+    pub submission_deadline_latest_block: u64,
```

Comment:
nit: maybe something a bit shorter. Also, in different places, this param is still named `submission_deadline`.
```suggestion
    pub deadline_block_num: u64,
```

Reply:
3 different naming nits from 3 people on the same variable name. 😂

Reply:
I didn't notice it in the resolved comments lol

Reply:
Ok guys @fleupold  @MartinquaXD @squadgazzz , let's discuss here the name, so we all agree on one :sweat_smile: 

So far the proposals are:

1) `submission_deadline_latest_block`
2) `deadline_block_num`
3) `deadline_block`

IMO, I think the shorter version name proposed is not as clear as the current one. I believe the current one (1) is the best of all of them.

Reply:
I'm fine with `1` since the slightly longer name is not used a lot anyway.